app/components/Layout/Footer.jsx

```diff
@@ -69,6 +76,23 @@ class Footer extends React.Component {
             nextState.showNodesPopup !== this.state.showNodesPopup
         );
     }
+    componentWillReceiveProps(recivedProps) {
```

Comment:
Can we use 'getDerivedStateFromProps' instead of this? 
`componentWillReceiveProps` is deprecated in latest React versions